File: EPSILON/TOOLS/Altova/Altova/XMLSpy2006/spl/cpp/AltovaText/EDISettings.cpp
[call GenerateFileHeader("EDISettings.cpp")]
#include "StdAfx.h"
#include "EDISettings.h"

namespace altova
{
namespace text
{
namespace edi
{

CEDISettings::CEDISettings()
:	m_TerminateSegmentsWithLinefeed(false)
,	m_AutoCompleteData(true)
{}

const CServiceStringAdvice& CEDISettings::GetServiceStringAdvice() const
{
	return m_UNA;
}
CServiceStringAdvice& CEDISettings::GetServiceStringAdvice()
{
	return m_UNA;
}
bool CEDISettings::GetTerminateSegmentsWithLinefeed() const
{
	return m_TerminateSegmentsWithLinefeed;
}
bool CEDISettings::GetAutoCompleteData() const
{
	return m_AutoCompleteData;
}
void CEDISettings::SetTerminateSegmentsWithLinefeed(bool rhs)
{
	m_TerminateSegmentsWithLinefeed= rhs;
}
void CEDISettings::SetAutoCompleteData(bool rhs)
{
	m_AutoCompleteData= rhs;
}


} // namespace edi
} // namespace text
} // namespace altova

File: EPSILON/SAMPLE FILE/E-publishing/Tools/Altova/Altova/XMLSpy2006/spl/java/altova/text/edifact/EDISettings.java
[call GenerateFileHeader("EDISettings.java")]
package com.altova.text.edifact;

public class EDISettings {
	private ServiceStringAdvice m_ServiceStringAdvice = new ServiceStringAdvice();

	private boolean m_TerminateSegmentsWithLinefeed = false;

	private boolean m_AutoCompleteData = true;

	public ServiceStringAdvice getServiceStringAdvice() {
		return m_ServiceStringAdvice;
	}

	public boolean getTerminateSegmentsWithLinefeed() {
		return m_TerminateSegmentsWithLinefeed;
	}

	public boolean getAutoCompleteData() {
		return m_AutoCompleteData;
	}

	public void setTerminateSegmentsWithLinefeed(boolean rhs) {
		m_TerminateSegmentsWithLinefeed = rhs;
	}

	public void setAutoCompleteData(boolean rhs) {
		m_AutoCompleteData = rhs;
	}
}
